course/src/main/java/in/testpress/course/services/VideoDownloadService.kt

```diff
@@ -83,9 +93,21 @@ class VideoDownloadService : DownloadService(
             Download.STATE_FAILED -> notification = getFailedNotification()
             Download.STATE_REMOVING -> downloadMonitor.deleteVideo(download)
         }
+
+        notification?.contentIntent = getNotifyPendingIntent()
```

Comment:
I could not understand what this variable refers to by seeing its name. Please change it to meaningful name.
Name should explain its intent

Reply:
Okay @Karthik-0, will change that.